tests/main/parallel-install-basic/task.yaml

```diff
@@ -0,0 +1,86 @@
+summary: Checks for parallel installation of a local snap files
+
+prepare: |
+    # ensure we have no snap user data directory yet
+    rm -rf /home/test/snap
+    rm -rf /var/snap/test-snapd-tools /var/snap/test-snapd-tools_foo
+
+execute: |
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+    path="$(make_snap test-snapd-tools)"
+
+    snap set system experimental.parallel-instances=true
+
+    devmode=
+    if [[ "$SPREAD_SYSTEM" = ubuntu-* ]]; then
+        # TODO prallel-install: disable devmode once AppArmor profile updates are in
+        devmode=--devmode
+    fi
+
+    snap install --dangerous --name test-snapd-tools "$path"
+    snap install --dangerous $devmode --name test-snapd-tools_foo "$path"
```

Comment:
Do you think we should have a helper for installing a local snap as an instance?

Reply:
I'll add it to my TODO and push it in a separate PR.

Reply:
Added the helper in #5774